app/importers/protege_importer.rb

```diff
@@ -0,0 +1,180 @@
+require 'open-uri'
+require 'zip'
+
+class ProtegeImporter
+
+  # This is the STAGING protege URL, will need to switch this to Production
+  ZIP_URL = "http://52.4.82.207:8080/webprotege/download?ontology=0aa09276-58a6-4350-a0bb-60eb2ab4be00"
+
+  def initialize(resource = ZIP_URL)
+    @resource = resource
+    @root_term = {}
+    @all_terms, @industry_terms, @industries, @sectors = [], [], [], []
+  end
+
+  def import
+    xml = extract_xml_from_zip
+    parse_terms_from_xml(xml)
+
+    @all_terms.each { |term| @industry_terms.push(term) if validate_industries_term(term) }
```

Comment:
BTW, you could do this like so: `@all_terms.concat(@industry_terms.select { |term| validate_industries_term(term) })`, which is slightly more "idiomatic".